plugins/topology/src/components/Topology/TopologySideBar/TopologyResourcesTabPanel.tsx

```diff
@@ -74,6 +77,43 @@ const TopologyResourcesTabPanel = ({
             </li>
           ))}
       </TopologyResourcesTabPanelItem>
+      {nodeData?.pipelinesData?.pipelines?.length && (
```

Comment:
will be nice to extract the value above and use it below as this is repeated

`const pipelinesData = nodeData?.pipelinesData?.pipelines`

and can we extract the pipelines code to separate component?